raiden/tests/fixtures/variables.py

```diff
@@ -92,12 +91,6 @@ def channels_per_node():
     return 1
 
 
-@pytest.fixture
-def poll_timeout():
```

Comment:
this means that either the tests will fail with the pytest-timeout or we need to use an explicit `gevent.Timeout`